### ISSUE TITLE
scripts.txt needs an update (I think)

### ISSUE THREAD

**Issue by LouDnl** [A3Wasteland/Release_Files#11]
BEServer.dll just got updated. Now getting all sorts of scripts.log entries that weren't there before.
Can't really find these scripts.log entries in the mission. These players are long time players on my server and never got logged before.

Dwarden said on Skype:

ye both BE client and server updated, same as in OA new scipts.txt logging any code size and performance optimization
it just logs, to activate the new filter (after you adjust it) to statr kicking again you must add line with
//new2
for start i suggest you first let it run in the logging mode to be sure it's not doing mess

log entries: http://arma.nl-united.nl/scriptslogs.7z

**Comment by LouDnl** [comment]
https://twitter.com/FoltynD/status/623842187239915520

**Comment by CRE4MPIE** [comment]
on a side note ...

I've set this to 5 , what are the chances here of false posetives ? I get maybe 1 - 2 per day ... Ban first, question later !

 1 "onMapSingleClick" !"\"onmapsingleclick\"" // log possible teleport hacks

**Comment by AgentRev** [comment]
I don't think there are false positives in vanilla, I set it to 1 in case some addons have a legitimate use for onMapSingleClick.

**Comment by LouDnl** [comment]
Scripts.txt isn't kicking for script use now. This gives script kiddies a free hand atm.

Also getting many false positives (I think). These are entries from the scripts.log just now.

23.07.2015 19:48:10: Dennis (92.111.48.245:2304) 3e2c29e73686cbdd2930f1c331d029a4 - #2 "e","g-e-f","box-esp","god on","god mode","unlimited mags","ctrlcreate","_execscript","_theban","rhynov1","b1g_b3n","e_x_t_a_s_y""
23.07.2015 20:07:01: Nobyz (86.88.204.139:2304) 0b66a49a527e3d0a3de2e870b9985fd2 - #2 "e","g-e-f","box-esp","god on","god mode","unlimited mags","ctrlcreate","_execscript","_theban","rhynov1","b1g_b3n","e_x_t_a_s_y""

**Comment by AgentRev** [comment]
That stuff is from infistar, are you running it?

**Comment by LouDnl** [comment]
No i'm not running it.

**Comment by LouDnl** [comment]
Also had a lot of these:

23.07.2015 01:57:12: PandaWithAnAKA.P.M.C 93dfa2649360b0d01d92765c330a1bf0 - #2 "getResolution select 2)1.1,(_0vn select 3)1.1];_0wn=_05m ctrlCreate["RscButtonMenu",-1,_07m];_0wn ctrlSetText "?";_0wn ctrlSet"

**Comment by AgentRev** [comment]
No idea what that is, but it looks obfuscated. I only know 3 major things that are obsfuscated in such a way: infiSTAR, Sa-Matra's missions, and Breaking Point. It could also be a hack, but I'm not sure.

**Comment by LouDnl** [comment]
Allright, i'll add them.

**Comment by AgentRev** [comment]
//new2 does not support regexp, no changes needed in that regard

**Comment by LouDnl** [comment]
Here's some new logs:

24.07.2015 18:33:10: luap995 (79.233.19.183:2304) 6167d4363fdd28c7f5b27ff126beaa19 - #4 " select 1),(_0vn select 3)*(getResolution select 3)/(getResolution select 2)*1.1,(_0vn select 3)*1.1];_0wn=_05m ctrlCreate["RscB"
24.07.2015 18:33:10: luap995 (79.233.19.183:2304) 6167d4363fdd28c7f5b27ff126beaa19 - #5 "ect 3)*1.1];_0wn=_05m ctrlCreate["RscButtonMenu",-1,_07m];_0wn ctrlSetText "?";_0wn ctrlSetPosition _pos;_0wn buttonSetAction fo"
24.07.2015 19:25:52: [Rekrut] Kirigaya (89.13.129.48:2304) 89fdaac364e2df5fd8c3fa4140cf30da - #4 " select 1),(_0vn select 3)*(getResolution select 3)/(getResolution select 2)*1.1,(_0vn select 3)*1.1];_0wn=_05m ctrlCreate["RscB"
24.07.2015 19:25:52: [Rekrut] Kirigaya (89.13.129.48:2304) 89fdaac364e2df5fd8c3fa4140cf30da - #5 "ect 3)*1.1];_0wn=_05m ctrlCreate["RscButtonMenu",-1,_07m];_0wn ctrlSetText "?";_0wn ctrlSetPosition _pos;_0wn buttonSetAction fo"
24.07.2015 20:39:42: Alcatrazz (84.155.184.70:2304) 5c65234b89d67bb04ea4284dcddc94da - #4 " select 1),(_0vn select 3)*(getResolution select 3)/(getResolution select 2)*1.1,(_0vn select 3)*1.1];_0wn=_05m ctrlCreate["RscB"
24.07.2015 20:39:42: Alcatrazz (84.155.184.70:2304) 5c65234b89d67bb04ea4284dcddc94da - #5 "ect 3)*1.1];_0wn=_05m ctrlCreate["RscButtonMenu",-1,_07m];_0wn ctrlSetText "?";_0wn ctrlSetPosition _pos;_0wn buttonSetAction fo"

**Comment by LouDnl** [comment]
[23:53:46] Alex Grosse: No don't ban
[23:54:07] Alex Grosse: That's from KOTG
[23:54:09] Alex Grosse: KOTH
[23:54:27] Alex Grosse: since memory isn't flushed when leaving servers

**Comment by AgentRev** [comment]
Yeah, that's weird. Anyway, I removed ctrlCreate from the filter, the command is being used more and more by the day for legit stuff, so exceptions would inevitably start to pile up in the long run.

**Comment by LouDnl** [comment]
Yea I still had those 2 lines added :) removed them now.
